### DISCUSSION TITLE
Newbie help

### DISCUSSION THREAD

**Discussion by rvxfahim** [carla-simulator/carla#5140] [Q&A]
I have been playing around with the example scripts recently.
Now I was thinking of implementing this scenario, but I am clueless. Seeking some guidance.
I will spawn a manually controlled car, and then I will spawn 4 more vehicles which will simply follow me. To my understanding all these NPCs will have a agent destination set to my vehicle's location. Based on some kind of trigger (key press / anything appropriate) I would ask these NPCs to stop or continue following me.
To make them stop following me, I suppose I should give them a different destination location for now. But I would like to ask NPCs to park in the nearest parking lane, for which I don't think there is any method to get a location of the nearest parking lane. 
Is this how I should proceed or is there any better methodology to do a platooning scenario.
Thanks in advance.

**Comment by ewencai** [comment]
这是来自QQ邮箱的自动回复邮件。您好，您的邮件我已经收到，我在看到后会尽快给您回复。
